mavros_msgs/msg/OpticalFlow.msg

```diff
@@ -0,0 +1,9 @@
+# OPTICAL_FLOW message data
+
+std_msgs/Header header
+
+int16[2] flow
+float32[2] flow_comp_m
+uint8 quality
+float32 ground_distance
+float32[2] flow_rate
```

Comment:
Usually ROS recommend to use full Vector3 and just don't set Z.

Reply:
Well, arrays looks worse than _x/_y.
But, why not to try this:
```
geometry_msgs/Vector3 flow
Vector3 flow_comp_m
Vector3 flow_rate
```

Reply:
flow in mavlink is int16_t, convert to float?

Reply:
I think yes, easier to use existing and well established type. We can define variety of Vector2, I16Vector2 and so, but for other ros users that anyway would need more and more conversions.